specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-10-02-preview/managedClusters.json

```
@@ -1348,6 +1348,64 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/agentPools/{agentPoolName}/deleteMachines": {
+      "post": {
+        "tags": [
+          "AgentPools"
+        ],
+        "operationId": "AgentPools_DeleteMachines",
+        "summary": "Deletes specific machines in an agent pool.",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AgentPoolNameParameter"
+          },
+          {
+            "name": "machines",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AgentPoolDeleteMachinesParameter"
+            },
+            "description": "A list of machines from the agent pool to be deleted."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Location": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
```

```suggestion
              "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
```

according to the [rule](https://github.com/Azure/azure-openapi-validator/blob/main/docs/lro-error-content.md), newly added lro error response should reuse the predefined model from common types.